ksqldb-functional-tests/src/test/resources/rest-query-validation-tests/pull-queries-against-streams.json

```diff
@@ -0,0 +1,137 @@
+  {
+  "comments": [
+    "Tests covering Pull queries of streams"
```

Comment:
@agavra , I added all the positive and negative cases I could think of here. Any I missed?